conf/char/char-server.conf

```diff
@@ -106,8 +106,8 @@ char_configuration: {
 			// Start point (Renewal)
 			start_point_re: {
 				map: "iz_int"
```

Comment:
The map changed because it was replaced, it is still iz_int. Above I provided a download for both iz_int maps